include/mbedtls/cmac.h

```diff
@@ -115,7 +119,9 @@ int mbedtls_cipher_cmac_update( mbedtls_cipher_context_t *ctx,
  *                      mbedtls_cipher_cmac_update(), or mbedtls_cipher_free().
  *
  * \param ctx           The cipher context used for the CMAC operation.
+ *                      This must be initialized.
  * \param output        The output buffer for the CMAC checksum result.
```

Comment:
We must document the required size of `output` here. To my understanding, it's the block-size of the underlying cipher.